src/database/migrations/20220822085453_add_field_notes_on_table_users.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex'
+
+const tableName = 'users'
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(tableName, (table) => {
+    table.text('notes')
```

Comment:
notes nya nullable ga mang @ayocodingit ?

Reply:
Kalau di knex itu secara defaultnya nullable mang 😁

Reply:
mantap atuh ya. kalau pengen set not null gimana?

Reply:
set gini kang  `table.text('notes').notNullable()` 🛩️

Reply:
kebalikannya migrate laravel ya wkwkw. nice insight kang. nuhun pisan